apps/aecore/src/aec_conductor.erl

```diff
@@ -404,6 +408,25 @@ get_beneficiary() ->
             {error, beneficiary_not_configured}
     end.
 
+set_stratum_mode(State) ->
+    case aeu_env:user_config_or_env([<<"stratum">>, <<"enabled">>], aecore, stratum_enabled) of
+        {ok, true} ->
+            {ok, Dir} = aeu_env:user_config_or_env(
+                          [<<"stratum">>, <<"reward">>, <<"keys">>, <<"dir">>],
+                          aecore, stratum_reward_keys_dir),
+            try aestratum_config:read_keys(Dir, create) of
+                {PubKey, _PrivKey} ->
+                    State#state{stratum_mode = true,
+                                beneficiary  = PubKey}
+            catch
+                _:_ ->
+                    %% stratum may be in setup phase which ends with restarting of node
```

Comment:
Could you please explicate.

Reply:
I've removed the try catch, it was a leftover.